app/components/m-header.hbs

```diff
@@ -24,7 +24,7 @@
           <Auk::Tab @route="cases" data-test-m-header-cases>
             {{t "cases"}}
           </Auk::Tab>
-          {{#if this.currentSession.isViewer}}
+          {{#if (user-may "view-newsletters")}}
```

Comment:
'Kort bestek' zou ik enkel zichtbaar maken voor gebruikers met rol `manage-newsletter-infos`. Ik weet dat dit minder groepen zijn dan in de huidige app. Maar net even getest, en die KB tab geeft toch errors voor andere profielen. Die tab heeft voor hen ook geen meerwaarde.